doc/modules/ensemble.rst

```diff
@@ -1345,18 +1343,18 @@ Usage
 
 The following example shows how to fit the VotingRegressor::
 
-   >>> from sklearn.datasets import load_boston
+   >>> from sklearn.datasets import load_diabetes
    >>> from sklearn.ensemble import GradientBoostingRegressor
    >>> from sklearn.ensemble import RandomForestRegressor
    >>> from sklearn.linear_model import LinearRegression
    >>> from sklearn.ensemble import VotingRegressor
 
    >>> # Loading some example data
-   >>> X, y = load_boston(return_X_y=True)
+   >>> X, y = load_diabetes(return_X_y=True)
 
    >>> # Training classifiers
-   >>> reg1 = GradientBoostingRegressor(random_state=1, n_estimators=10)
-   >>> reg2 = RandomForestRegressor(random_state=1, n_estimators=10)
```

Comment:
I am +0.5 on this change. I think the original intention of setting `n_estimators` is to reduce the runtime of doctest.

Reply:
No problem, I will amend here and in the corresponding example (https://github.com/scikit-learn/scikit-learn/pull/16895#issuecomment-612399378)

Reply:
I am so sorry for being unclear.

I do prefer removing `n_estimators` here as long as the increase in doctest runtime is not too great. In this case, the `fit` runtime is increased by 1~ second, which is acceptable.

Reply:
Okay I will change it back!